### PR TITLE
Rename phases

### PR DESCRIPTION
**Pull request by sebratton**
Internal phase names have changed to BEFORE_APP_START, AFTER_APP_START

### CODE REVIEW

**Comment by sebratton** [comment]
#BUILD

**Comment by sebratton** [comment]
#build

File: dev/com.ibm.ws.ejbcontainer/src/com/ibm/ws/ejbcontainer/osgi/internal/EJBRuntimeImpl.java [hunk 1]
```diff
@@ -1722,7 +1722,7 @@ protected void unsetLibertyFeature(ServiceReference<LibertyFeature> feature) {
      */
     @Override
     public boolean isCheckpointDeployment() {
```
Comment (tjwatson) [inline dev/com.ibm.ws.ejbcontainer/src/com/ibm/ws/ejbcontainer/osgi/internal/EJBRuntimeImpl.java]: We should get with @tkburroughs about changing this method name.

File: dev/com.ibm.ws.ejbcontainer/src/com/ibm/ws/ejbcontainer/osgi/internal/EJBRuntimeImpl.java [hunk 2]
```diff
@@ -1733,7 +1733,7 @@ public boolean isCheckpointDeployment() {
      */
     @Override
     public boolean isCheckpointApplications() {
```
Comment (tjwatson) [inline dev/com.ibm.ws.ejbcontainer/src/com/ibm/ws/ejbcontainer/osgi/internal/EJBRuntimeImpl.java]: Same comment about changing method name.

File: dev/com.ibm.ws.jpa.tests.container.checkpoint_fat.common/fat/src/com/ibm/ws/jpa/tests/container/checkpoint/tests/JPADataSourceCheckpointTest_EJB.java [hunk 3]
```diff
@@ -215,104 +215,104 @@ public void jpa_INACTIVE_testCheckpointDataSource_SL_CMTS_EJB() throws Exception
         testCheckpointDataSource(EJB_SL_SERVLET_NAME, phase, resourceType);
     }
 
-    // testing for CheckpointPhase.DEPLOYMENT
+    // testing for CheckpointPhase.BEFORE_APP_START
 
     @Test
     public void jpa_DEPLOYMENT_testCheckpointDataSource_SF_AMJTA_EJB() throws Exception {
```
Comment (tjwatson) [inline dev/com.ibm.ws.jpa.tests.container.checkpoint_fat.common/fat/src/com/ibm/ws/jpa/tests/container/checkpoint/tests/JPADataSourceCheckpointTest_EJB.java]: Probably should change the test names here.

File: dev/com.ibm.ws.jpa.tests.container.checkpoint_fat.common/fat/src/com/ibm/ws/jpa/tests/container/checkpoint/tests/JPADataSourceCheckpointTest_Web.java [hunk 4]
```diff
@@ -180,48 +180,48 @@ public void jpa_INACTIVE_testCheckpointDataSource_CMTS_Web() throws Exception {
         testCheckpointDataSource(phase, resourceType);
     }
 
-    // testing for CheckpointPhase.DEPLOYMENT
+    // testing for CheckpointPhase.BEFORE_APP_START
 
     @Test
     public void jpa_DEPLOYMENT_testCheckpointDataSource_AMJTA_Web() throws Exception {
```
Comment (tjwatson) [inline dev/com.ibm.ws.jpa.tests.container.checkpoint_fat.common/fat/src/com/ibm/ws/jpa/tests/container/checkpoint/tests/JPADataSourceCheckpointTest_Web.java]: Tests name change.

File: dev/io.openliberty.checkpoint/test/io/openliberty/checkpoint/internal/CheckpointImplTest.java [hunk 5]
```diff
@@ -400,7 +400,7 @@ public void testCheckpointApplications() throws CheckpointFailedException {
         TestCheckpointHook hook = new TestCheckpointHook(criu.singleThreaded);
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint/test/io/openliberty/checkpoint/internal/CheckpointImplTest.java:400]: Test name change.

File: dev/io.openliberty.checkpoint/test/io/openliberty/checkpoint/internal/CheckpointImplTest.java [hunk 6]
```diff
@@ -412,7 +412,7 @@ public void testCheckpointDeployment() throws CheckpointFailedException, Illegal
...
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint/test/io/openliberty/checkpoint/internal/CheckpointImplTest.java:412]: Test name change

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/BellsTest.java [hunk 7]
```diff
@@ -76,13 +76,13 @@ public void setUp() throws Exception {
         testMethod = getTestMethod(TestMethod.class, testName);
         switch (testMethod) {
             case testBellsCheckpointAtDeployment:
-                server.setCheckpoint(CheckpointPhase.DEPLOYMENT, false, null);
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/BellsTest.java:79]: Test method name change.

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/BellsTest.java [hunk 8]
```diff
@@ -76,13 +76,13 @@ public void setUp() throws Exception {
...
+                server.setCheckpoint(CheckpointPhase.BEFORE_APP_START, false, null);
                 break;
             case testBellsCheckpointAtApplication:
-                server.setCheckpoint(CheckpointPhase.APPLICATIONS, false, null);
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/BellsTest.java:82]: Test method name change.

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/CheckpointPhaseTest.java [hunk 9]
```diff
@@ -82,7 +82,7 @@ public void testAtApplicationsMultRestore() throws Exception {
 
     @Test
     public void testAtDeployment() throws Exception {
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/CheckpointPhaseTest.java]: Test name change.

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/SlowAppStartTest.java [hunk 10]
```diff
@@ -58,7 +58,7 @@ public static void setUpClass() throws Exception {
 
     @Test
     public void testAtDeployment() throws Exception {
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/SlowAppStartTest.java]: change test name

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/WebAppMessageTest.java [hunk 11]
```diff
@@ -58,20 +58,20 @@ public static void copyAppToDropins() throws Exception {
 
     @Test
     public void testApplicationsWebAppMessage() throws Exception {
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/WebAppMessageTest.java]: change method name

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/WebAppMessageTest.java [hunk 12]
```diff
@@ -58,20 +58,20 @@ public static void copyAppToDropins() throws Exception {
...
-        doTest(CheckpointPhase.APPLICATIONS);
+        doTest(CheckpointPhase.AFTER_APP_START);
     }
 
     @Test
     public void testDeploymentWebAppMessage() throws Exception {
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/WebAppMessageTest.java]: change method name

File: dev/io.openliberty.ejbcontainer.checkpoint_fat/fat/src/io/openliberty/ejbcontainer/checkpoint/fat/tests/EjbStartCheckpointTest.java [hunk 13]
```diff
@@ -164,7 +164,7 @@ public void testEjbStartCheckpointInactiveStartAllSingletons() throws Exception
 
     @Test
     public void testEjbStartCheckpointDeployment() throws Exception {
```
Comment (tjwatson) [inline dev/io.openliberty.ejbcontainer.checkpoint_fat/fat/src/io/openliberty/ejbcontainer/checkpoint/fat/tests/EjbStartCheckpointTest.java]: test method name

File: dev/io.openliberty.ejbcontainer.checkpoint_fat/fat/src/io/openliberty/ejbcontainer/checkpoint/fat/tests/EjbStartCheckpointTest.java [hunk 14]
```diff
@@ -178,7 +178,7 @@ public void testEjbStartCheckpointDeployment() throws Exception {
 
     @Test
     public void testEjbStartCheckpointApplications() throws Exception {
```
Comment (tjwatson) [inline dev/io.openliberty.ejbcontainer.checkpoint_fat/fat/src/io/openliberty/ejbcontainer/checkpoint/fat/tests/EjbStartCheckpointTest.java]: test method name

**Review by tjwatson — APPROVED** [review]
I have some follow on comments about renaming methods and tests.  That can be done in another PR.

**Review by tjwatson — DISMISSED** [review]
I have some follow on comments about renaming methods and tests.  That can be done in another PR.

File: dev/com.ibm.ws.ejbcontainer/src/com/ibm/ws/ejbcontainer/osgi/internal/EJBRuntimeImpl.java (hunk 1 above)
Comment (sebratton) [reply dev/com.ibm.ws.ejbcontainer/src/com/ibm/ws/ejbcontainer/osgi/internal/EJBRuntimeImpl.java]: Will change to:
  public boolean isCheckpointBeforeAppStart()
  public boolean isCheckpointAfterAppStart() 
I asked @tkburroughs on a different channel to let us know if he has another preference.

File: dev/com.ibm.ws.jpa.tests.container.checkpoint_fat.common/fat/src/com/ibm/ws/jpa/tests/container/checkpoint/tests/JPADataSourceCheckpointTest_EJB.java (hunk 3 above)
Comment (sebratton) [reply dev/com.ibm.ws.jpa.tests.container.checkpoint_fat.common/fat/src/com/ibm/ws/jpa/tests/container/checkpoint/tests/JPADataSourceCheckpointTest_EJB.java]: done

File: dev/com.ibm.ws.ejbcontainer/src/com/ibm/ws/ejbcontainer/osgi/internal/EJBRuntimeImpl.java (hunk 1 above)
Comment (tkburroughs) [reply dev/com.ibm.ws.ejbcontainer/src/com/ibm/ws/ejbcontainer/osgi/internal/EJBRuntimeImpl.java]: New names look good.

**Review by tjwatson — DISMISSED** [review]

**Review by tkburroughs — DISMISSED** [review]

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#libby

**Review by tkburroughs — DISMISSED** [review]

**Review by tjwatson — DISMISSED** [review]

**Comment by sebratton** [comment]
#build
#libby

**Comment by sebratton** [comment]
#build

**Review by tjwatson — APPROVED** [review]

**Comment by sebratton** [comment]
Only failues in PB were known issues or related to network infrastructure

**Pull request merged**